Mathlib/Data/Nat/Prime/Basic.lean

```diff
@@ -37,6 +37,9 @@ theorem Prime.dvd_iff_eq {p a : ℕ} (hp : p.Prime) (a1 : a ≠ 1) : a ∣ p ↔
   · exact mul_one _
   · exact (a1 rfl).elim
 
+theorem Prime.coprime_iff_ne {p q : ℕ} (hp : p.Prime) (hq : q.Prime) : p.Coprime q ↔ p ≠ q := by
```

Comment:
Isn't this exactly `Nat.coprime_primes`?

Reply:
I'll just close this PR.